app/models/FileUploadRequest.scala

```diff
@@ -30,7 +30,7 @@ final case class FileUploadRequest(declarationId: MRN, files: Seq[FileUploadFile
     </FileUploadRequest>
 }
 
-sealed abstract case class FileUploadFile(fileSequenceNo: Int, documentType: String, serviceUrl: String) {
+sealed case class FileUploadFile(fileSequenceNo: Int, documentType: String, serviceUrl: String) {
```

Comment:
Not sure about the purpose of `sealed` keyword here. `case class` should not be extended anyway.